Source/DafnyLanguageServer/README.md

```diff
@@ -96,156 +96,4 @@ Options provided through the command line have higher priority than the options
 # For just the dafny executable, replace `--dafny:plugins:X` by `--plugin:` that you can repeat.
 ```
 
-#### About plugins
```

Comment:
I'm guessing that this is the text that's now in `Plugins.md`, but that file seems to be missing from this PR.

Reply:
Ah yes ... git add ...